src/auxinfo/participant.rs

```diff
@@ -497,7 +497,7 @@ fn new_auxinfo<R: RngCore + CryptoRng>(
     debug!("Creating new auxinfo.");
 
     let (decryption_key, p, q) = DecryptionKey::new(rng)?;
-    let params = VerifiedRingPedersen::extract(&decryption_key, rng)?;
+    let params = VerifiedRingPedersen::extract(&decryption_key, &(), rng)?;
```

Comment:
todo: The `new_auxinfo` function should probably be inside the `impl AuxInfoParticipant` block instead. That way we can call `self.retrieve_context()` instead of instantiating context or taking it as a parameter here.

Reply:
done.